datalad/interface/crawl_init.py

```diff
@@ -76,6 +76,8 @@ def __call__(args=None, template=None, template_func=None, commit=False):
                 raise ValueError(
                     "args entered must be given in a list or dict, were given as %s",
                     type(args))
+        elif not template and not template_func:
+            raise TypeError("crawl-init takes a minimum of 2 arguments (0 given)")
```

Comment:
also please fix docstring for template_func -- it is not a flag, it is a name of the function